inc/class-saml.php

```diff
@@ -89,13 +91,22 @@ class SAML {
 	 */
 	private $samlSettings = [];
 
+	/**
+	 * @var Log
+	 */
+	private $log;
+
 	/**
 	 * @return SAML
 	 */
 	static public function init() {
 		if ( is_null( self::$instance ) ) {
 			$admin = Admin::init();
-			self::$instance = new self( $admin );
+			$log = null;
+			if ( CloudWatchProvider::areEnvironmentVariablesPresent() ) {
```

Comment:
Log module will be ignored if environment variables for CloudWatch are not present.